geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -141,6 +141,11 @@
             <artifactId>scala-compiler</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <scope>provided</scope>
```

Comment:
Is this added to remove commons-text from the distribution?

Reply:
yeah, it ships with accumulo 2 now, so we don't need to bundle it into the lib/ folder. I added it to the install-hadoop-accumulo.sh script for the use case of not having an ACCUMULO_HOME set. I don't think we're using it directly, as things seemed to still work ok with accumulo 1.9 which doesn't ship it. we should definitely do some thorough testing of different installs/versions once everything has stabilized.

Reply:
Wow, that's a lot of considerations in a short span.  Can I put you on the hook for indicating a few test environments / use cases that'd cover our bases?

Reply:
for the CLI tools i think the matrix looks like:
accumulo version: (1.7, 1.8)?, 1.9, 2.0
accumulo jars: ACCUMULO_HOME + HADOOP_HOME, install-hadoop-accumulo.sh
I ran through those 4 combos (for accumulo >= 1.9) with ingest, distributed ingest, export, distributed export and spark (although i didn't test distributed processing without HADOOP_HOME, as you need hadoop installed for that...)
for geoserver i used install-hadoop-accumulo.

Reply:
@khobbs-ccri can you capture this information in your V&V test plan?

Reply:
Wow, the e-mail summary was cryptic!

Reply:
@khobbs-ccri lemme know when you've got these pearls recorded!

Reply:
done